client/components/jetpack-logo/docs/example.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import JetpackLogo from 'components/jetpack-logo';
+
+export default function JetpackLogoExample() {
+	return (
+		<div>
+			<div>
+				<p>
+					<code>
+						{ '<JetpackLogo full />' }
+					</code>
+				</p>
+				<JetpackLogo full />
+			</div>
+			<div>
+				<p>
+					<code>
```

Comment:
I don't think we need the `<code>` part in the example. It would be more useful if we can get that one automatically for each component, rather than manually having it in the example.

Reply:
👍  2099e828bb0bcea8d60c1b97c73e907a897a57f2